docs/current/sdk/go/148388-guides.md

```diff
@@ -0,0 +1,9 @@
+---
+slug: /sdk/go/148388/guides
+---
+
+# Guides
+
+## Core Concepts
```

Comment:
I don't think we need to categorize at this point. There isn't enough content yet.

Reply:
Similarly, I would place the content file in guides/, removing the core-concepts directory (at least for now).

Reply:
Agree, keep the structure simple. CUE SDK has way too much structure, makes it hard to navigate.